src/draw/draw.h

```diff
@@ -4,6 +4,7 @@
 
 #include <stdint.h>
 #include <stddef.h>
+#include <draw.h>
```

Comment:
This is a dumb workaround to having 2 headers called "draw.h" flagging a linter error

I need to rename my headers, I don't love the linter rule but I think 2x draw.h is confusing